Assets/Scripts/HPPickupInteract.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+public class HPPickupInteract : MonoBehaviour
+{
+    // Private Variables \\
```

Comment:
I think the comments in this script are unnecessary.

Reply:
Personal style, I'll remove them though.